engine/core.go

```diff
@@ -116,6 +116,15 @@ func NewEngine(cfg *config.EngineConfig) *Engine {
 	return engine
 }
 
+// NewEngine returns an initialised Engine struct.
+func NewEngine(cfg *config.EngineConfig) *Engine {
+	ncc, err := ncclient.NewNCC(cfg.NCCSocket)
+	if err != nil {
+		log.Fatalf("failed to create ncc client: %v", err)
```

Comment:
Be consistent with other logs - log.Fatalf("Failed to create NCC client: %v", err)

Reply:
done